thrift/src/test/java/com/linecorp/armeria/server/thrift/ThriftDocServiceTest.java

```diff
@@ -79,7 +79,7 @@
 
     private static final ObjectMapper mapper = new ObjectMapper();
 
-    private static final boolean DEMO_MODE = false;
+    private static final boolean DEMO_MODE = true;
```

Comment:
Would you please change this to `false` as before? Because this may block builds on our CI. :-)